### ISSUE TITLE
Visually indicate when a plot has invalid code

### ISSUE THREAD

**Issue by TheBizzle** [NetLogo/Galapagos#308]
When you click a button that contains bad code (including prims that don't yet compile), they don't indicate that they're invalid, and they throw an error on use.  Instead, they should be unclickable, and they should show specifically that the code is invalid (like how the text gets red in JVM NetLogo).

**Comment by nicolaspayette** [comment]
Plots with non-compilable code should probably give some visual indication of that too.

**Comment by TheBizzle** [comment]
Agreed.  I wasn't clear about this when I originally wrote the ticket, but any widget with invalid code (slider, plot, button, monitor, chooser) should be shown as broken.

**Comment by TheBizzle** [comment]
With the introduction of c435db994cf7e4384927c2ca534fd45a6f27e0bc, this issue has been solved for all widgets but plots.

**Comment by LaCuneta** [comment]
If a plot has a runtime error it is squelched in the `runWithErrorHandling()` method from `set-up-widgets.coffee` to avoid interrupting the model.  It should be displayed to the user in some non-intrusive way as it would be in NetLogo desktop.  This is slightly different than the compile time error display, but I think it makes sense to address both at the same time.
